### DISCUSSION TITLE
Include website repository in Streampipes main repository

### DISCUSSION THREAD

**Discussion by bossenti** [apache/streampipes#965] [General]
I would like to discuss a (maybe wild) idea that came up to my mind: Do we want to integrate our `streampipes-website` repository containing or docs and website into our main repository as well?
Since we are already working with a mono-repo approach in the main repository, this seems like a logical step for me.
This would have a lot of benefits for keeping our code docs and references up to date, reducing overhead, etc.

Are there any objections?
Have you thought about this in the past?
Was there any reason to not include this repository when we integrated the `extensions` and `installer` repos into the main repository?

**Comment by tenthe** [comment]
Hey @bossenti, I am in favor of your suggestion. 
I think it makes sense to have all parts of the project in one repository.


I think we need to find out if there are any problems with the configuration of the automatic deployment.
However, I think that this topic is not very urgent. So my suggestion would be to create an issue and mark it as a "good first issue". I think it could be a good starting point for new developers to get familiar with the code base.
What do you think?

**Reply by bossenti** [reply]
Sure this is for sure nothing we need to tackle right now.
I'm not sure whether this a bit large for a good first issue?
